src/core_modules/capture-core/reducers/descriptions/currentSelections.reducerDescriptionGetter.js

```diff
@@ -216,9 +217,20 @@ export const getCurrentSelectionsReducerDesc = (appUpdaters: Updaters) => create
         categories: undefined,
         categoriesMeta: undefined,
     }),
-    [searchPageActionTypes.FALLBACK_SEARCH_COMPLETED]: (state, { payload: { trackedEntityTypeId } }) => ({
+    [searchPageActionTypes.FALLBACK_SEARCH_COMPLETED]:
+      (state, { payload: { trackedEntityTypeId } }) => ({
+          ...state,
+          programId: undefined,
+          categories: undefined,
+          categoriesMeta: undefined,
```

Comment:
Thought is a good idea to clean those when your selection is a `trackedEntityTypeId`